docs/en/observability/ci-cd-observability.asciidoc

```diff
@@ -427,3 +427,50 @@ The Concourse CI doesn’t report health metrics through OpenTelemetry. However,
 https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/spanmetricsprocessor#span-metrics-processor[OpenTelemetry Collector Span Metrics Processor]
 to derive pipeline execution traces into KPI metrics like throughput and the error rate
 of pipelines.
+
+
+[discrete]
+[[check-service-health-from-cd-pipelines]]
+== Check service health from deployment pipelines
+
+
+Integrating automated service health checks in deployment pipelines is critical to end-to-end deployment automation which is critical to increase the deployment frequency .
```

Comment:
There are two `critical` in the same sentence, what about:

 ```suggestion
Integrating automated service health checks in deployment pipelines is critical to end-to-end deployment automation which is fundamental to increase the deployment frequency.
```

Reply:
@ollyhowell can we defer to you and tech writers on @v1v 's proposal "fundamental" vs "critical". I like it but i'm not a specialist, English is not my mother tongue.

Reply:
```suggestion
Integrating automated service health checks in deployment pipelines is critical for end-to-end deployment automation, which crucially enables deployment frequency to be increased.
```

Thanks for tagging me - ask a writer and you should always get feedback or an alternative suggestion :) how about this?

Reply:
@v1v are you ok if we pick the proposal of our tech writer? :-)